src/main/java/org/trips/service_framework/dtos/SkuAttributes.java

```diff
@@ -34,12 +33,13 @@ public class SkuAttributes {
     private String unitPerCarton;
     private String catchType;
     private String grade;
+    private String certification;
 
     @AssertTrue(message = "Quantity information format is not correct")
     public boolean isOk() {
-        return (Objects.nonNull(unitPerCarton) && Objects.nonNull(quantityPerUnit) && Objects.isNull(unitWeight)) ||
-                (Objects.isNull(unitPerCarton) && Objects.isNull(quantityPerUnit) && Objects.nonNull(unitWeight)) ||
-                (Objects.isNull(unitPerCarton) && Objects.isNull(quantityPerUnit) && Objects.isNull(unitWeight));
+        return (CmsUtils.nonNull(unitPerCarton) && CmsUtils.nonNull(quantityPerUnit) && CmsUtils.isNull(unitWeight)) ||
+                (CmsUtils.isNull(unitPerCarton) && CmsUtils.isNull(quantityPerUnit) && CmsUtils.nonNull(unitWeight)) ||
+                (CmsUtils.isNull(unitPerCarton) && CmsUtils.isNull(quantityPerUnit) && CmsUtils.isNull(unitWeight));
```

Comment:
in which scenario can we have all 3 of them null?

Reply:
Assume we create an SKU during Inward or grading step where unit per carton, quantity per unit and unit weight not present